src/sage/functions/special.py

```diff
@@ -455,9 +455,8 @@ class EllipticE(BuiltinFunction):
         sage: z = var("z")
         sage: elliptic_e(z, 1)
         elliptic_e(z, 1)
-        sage: # this is still wrong: must be abs(sin(z)) + 2*round(z/pi)
-        sage: elliptic_e(z, 1).simplify()
-        2*round(z/pi) + sin(z)
+        sage: elliptic_e(z, 1).simplify() # not tested - gives wrong answer with maxima < 5.47
```

Comment:
I guess this is OK for now. This is testing for a specific maxima bug (#3967) that was fixed in 5.47, and that is already part of the maxima test suite. So normally, I would say "delete it," but there aren't many other examples for `elliptic_e` and showing that it can be simplified to a more-explicit form is nice.